src/components/charts/ChartHeaderFilters.tsx

```diff
@@ -60,50 +65,73 @@ function ChartHeaderFilter({ selectedChartRepo, handleCloseFilter, includeDeprec
-            {searchApplied ? <button className="search__clear-button" type="button" onClick={(e) => handleFilterChanges(e, "clear")}>
-                <Clear className="icon-dim-18 icon-n4 vertical-align-middle" />
-            </button> : null}
-        </form>
-        <div className="flex">
-            <ReactSelect
-                className="date-align-left fs-13"
-                placeholder="Repository : All"
-                name="repository"
-                value={selectedChartRepo}
-                options={chartRepoList}
-                onChange={setSelectedChartRepo}
-                isClearable={false}
-                isMulti={true}
-                closeMenuOnSelect={false}
-                hideSelectedOptions={false}
-                onMenuClose={handleCloseFilter}
-                components={{
-                    DropdownIndicator,
-                    Option,
-                    ValueContainer,
-                    IndicatorSeparator: null,
-                    ClearIndicator: null,
-                    MenuList,
-                }}
-                styles={{ ...multiSelectStyles }} />
-            <Checkbox rootClassName="ml-16 mb-0 fs-14 cursor bcn-0 pt-8 pb-8 pr-12 date-align-left--deprecate"
-                isChecked={includeDeprecated === 1}
-                value={"CHECKED"}
-                onChange={(event) => { let value = (includeDeprecated + 1) % 2; handleFilterChanges(value, "deprecated") }}>
-                <div className="ml-5"> Show deprecated</div>
-            </Checkbox>
+   
+    return (
+        <div className="filter-column-container">
+            <div className="pb-12 pl-12 pr-12 pt-16">
+                <form onSubmit={(e) => handleFilterChanges(e, 'search')} className="search-column position-rel">
+                    <Search className="search__icon icon-dim-18" />
+                    <input
+                        type="text"
+                        placeholder="Search charts"
+                        value={appStoreName}
+                        className="search__input bcn-0"
+                        onChange={(event) => {
+                            setAppStoreName(event.target.value)
+                        }}
+                    />
+                    {searchApplied ? (
+                        <button
+                            className="search__clear-button"
+                            type="button"
+                            onClick={(e) => handleFilterChanges(e, 'clear')}
+                        >
+                            <Clear className="icon-dim-18 icon-n4 vertical-align-middle" />
+                        </button>
+                    ) : null}
+                </form>
+            </div>
+            <div className="pl-12 pr-12 filter-tab">
+                <div className="fs-12 fw-6 ml-8 cn-6 pb-8 pt-8">VIEW AS</div>
+                <div className="cursor">
+                    <div
+                        onClick={() => setGrid(true)}
```

Comment:
Please set the grid value in a separate function and call it here in onclick

Reply:
Done